TVRemoteControl/ScannerTableViewController.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  ScannerTableViewController.swift
+//  TVRemoteControl
+//
+//  Created by Daian Aiziatov on 26/02/2019.
+//  Copyright © 2019 robot64. All rights reserved.
+//
+
+import UIKit
+
+class ScannerTableViewController: UITableViewController {
+
+    private lazy var progressView: UIProgressView = {
+        let frames = CGRect(x: 0,
+                            y: (self.navigationController?.navigationBar.frame.height ?? 0) + 20,
+                            width: (self.navigationController?.navigationBar.frame.width ?? 0),
+                            height: 20)
+        let progressView = UIProgressView(frame: frames)
+        progressView.translatesAutoresizingMaskIntoConstraints = false
+        return progressView
+    }()
+
+    private var connectedTVs = [SystemInfo]()
```

Comment:
According to popular Swift style guides, `var connectedTVs: [SystemInfo] = []` looks more usual

[example](https://github.com/raywenderlich/swift-style-guide)

Reply:
Linted